src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,70 @@
+package core.basesyntax;
+
+import core.basesyntax.dao.fileReader;
+import core.basesyntax.dao.fileReaderImpl;
+import core.basesyntax.dao.writeReport;
+import core.basesyntax.dao.writeReportImpl;
```

Comment:
The class names `fileReader`, `fileReaderImpl`, `writeReport`, and `writeReportImpl` do not follow Java naming conventions. Class names should start with an uppercase letter, e.g., `FileReader`, `FileReaderImpl`, `WriteReport`, and `WriteReportImpl`.